package.json

```diff
@@ -64,28 +64,29 @@
     "es6-shim": "^0.35.1",
     "ghooks": "^1.3.2",
     "gkatsev-grunt-sass": "^1.1.1",
-    "grunt": "^0.4.4",
+    "grunt": "^0.4.5",
```

Comment:
grunt has to remain in 0.4.x because `grunt-github-releaser` requires it as a peer dependency.